src/tox_uv/_venv.py

```diff
@@ -83,6 +86,18 @@ def _get_python(self, base_python: list[str]) -> PythonInfo | None:  # noqa: PLR
                     platform=sys.platform,
                     extra={},
                 )
+            if os.path.isabs(base):  # noqa: PTH117
+                info = cached_py_info.from_exe(
+                    cached_py_info.PythonInfo, app_data.make_app_data(None, read_only=False, env=os.environ), base
+                )
+                return PythonInfo(
+                    implementation=info.implementation,
+                    version_info=VersionInfo(*info.version_info),
+                    version=info.version,
+                    is_64=info.architecture >= 64,  # noqa: PLR2004
```

Comment:
should be == not >=

Reply:
Interesting -- means that the `sys.maxsize > 2**32` comparison seen elsewhere is too permissive? (will also catch 128-bit, for example)

I've changed it anyhow